perfkitbenchmarker/pkb.py

```diff
@@ -195,35 +258,46 @@ def RunBenchmark(benchmark, collector, sequence_number, total_benchmarks):
         logging.exception('Prerequisite check failed for %s', benchmark_name)
         raise
 
-    start_time = time.time()
+    # Initialize timed interval objects.
+    end_to_end_interval = timed_interval.TimedInterval('End to End')
+    prepare_phase_interval = timed_interval.TimedInterval('Prepare Phase')
```

Comment:
Recommend converting "Prepare Phase" to "Resource Provisioning", exclusive of "Benchmark Prepare" time.